vm/src/obj/objset.rs

```diff
@@ -67,11 +67,21 @@ impl PyValue for PyFrozenSet {
 }
 
 #[derive(Default, Clone)]
-struct PySetInner {
+pub struct PySetInner {
```

Comment:
Do we realy need to expose this? This is used only once to create a `frozenset` (unless I missed something)

Reply:
I'm not sure, I guess I could just have some sort of `from_iter` function for PyFrozenSet.